src/test/java/com/kodilla/ecommercee/domainTests/CartTestSuite.java

```diff
@@ -118,27 +109,20 @@ public void testAddProductToCart() {
 
```

Comment:
nie wiem o co chodzi, ale sugestie mi nie działają tak jak chcę. tu chodzi o linijki 107-108. wydaje mi się, że to upraszcza sprawę

```suggestion
        cartRepository.save(cart);
        Optional<Cart> retrievedCart = cartRepository.findById(cart.getId());
```

Reply:
Czyli wtedy pozmieniać _Optional<Cart> retrievedCart = cartRepository.findById(savedCart.getId());_ na _Optional<Cart> retrievedCart = cartRepository.findById(cart.getId());_

Będzie bardziej spójnie. ; p

Reply:
o to to :D